### DISCUSSION TITLE
web_sys Response is opaque.

### DISCUSSION THREAD

**Discussion by greenpdx** [wasm-bindgen/wasm-bindgen#4296] [Q&A]
### Summary

I copied the web_sys fetch example and only changed the URL and the Accept to "application/json" and console::log_1() statements 
I am running the WASM inside a VueJS application in a Firefox.  I see the json data in the network debugger.

right after the "let resp_value = ....." the result is an opaque response.

Why is this and how do I fix it?

**Comment by daxpedda** [comment]
Probably a **minimal** reproducable example and screenshots explaining what exactly you are seeing would go a long way to let others help you.

However, you might want to ask in a more general Web community, as I don't think this is a `wasm-bindgen` issue. But let me know if otherwise!
